src/vehicleUrban.cpp

```diff
@@ -95,10 +105,45 @@ void socCallbackBms(float soc, float voltage) {
 	canInterface.sendMessage(message);
 }
 
+// Set Bms Remotely
```

Comment:
Is there any way we can make this automatic? For example, the abstract class could just grab the latest data from whichever BMS is connected? It will make this much easier during competition when switching batteries.